### PR TITLE
Fix "Rendered more hooks than during the previous render." in LineGraph

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

I was getting these errors occasionally within `LineGraph`:
Screen Shot 2022-01-27 at 16 43 08

It appears the cause is `LineGraph` returning early (i.e. before all hooks run), which is prone to some race condition causing it NOT to return early at times, in which case all the V2 hooks get to run and make React extremely confused. The solution is to just refactor the `LineGraph` version-choosing logic into a component of it's own that just renders the right _actual_ component.

### CODE REVIEW

**Comment by Twixes** [comment]
Hmm, looks like there are two crashes possible:
1. The feature flags case, fixed
2. A bug in the`chartjs-plugin-crosshair` https://github.com/AbelHeinsbroek/chartjs-plugin-crosshair/issues/10, not sure how to fix yet

**Comment by Twixes** [comment]
I believe problem 2. is a race condition between when the chart is generated and when it's sized, but not sure where exactly does this occur. However, this seems to occur only in `InsightCard` AND appears to be fixed by #8302 – specifically, the new `InsightViz` sizing mechanism. Therefore this PR is viable on its own.

File: frontend/src/scenes/insights/LineGraph/LineGraph.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import {
     TooltipModel,
     TooltipOptions,
 } from 'chart.js'
-import { CrosshairOptions, CrosshairPlugin } from 'chartjs-plugin-crosshair'
+import CrosshairPlugin, { CrosshairOptions } from 'chartjs-plugin-crosshair'
```
Comment (alexkim205) [inline frontend/src/scenes/insights/LineGraph/LineGraph.tsx:20]: Oo good catch

**Review by alexkim205 — APPROVED** [review]
Thanks for fixing this :) this works great

**Pull request merged**